wagtailnhsukfrontend/blocks.py

```diff
@@ -49,3 +49,12 @@ class InsetTextBlock(RichTextBlock):
 
     class Meta:
         template = 'wagtailnhsukfrontend/inset_text.html'
+
+
+class BackLinkBlock(StructBlock):
+
+    link_title = CharBlock(default="Go Back", label="Go Back", required=True)
+    back_url = URLBlock(label="Enter the back URL", required=True)
```

Comment:
Maybe we should have a page chooser block here?
I'm not convinced of the need for a BackLinkBlock anyway.